code/modules/cargo/packs/medical.dm

```diff
@@ -102,6 +101,75 @@
 		var/item = pick(contains)
 		new item(C)
 
+/datum/supply_pack/medical/medkits
```

Comment:
This is a general revivew, but move the crates to the monkestation folder please